manage.py

```diff
@@ -287,6 +287,30 @@ def refresh_continuous_aggregates():
     """
     ts_refresh_listen_count_aggregate()
 
+@cli.command()
+@click.option("-u", "--user", type=str)
+@click.option("-t", "--token", type=str)
+@click.argument("releaseid", type=str)
```

Comment:
minor nitpick: Given how many IDs we have floating about, I've started using the convention of being explicit about what type of id it is. Now I always try to use: 

- <entity>_id : for row ids
- <entity>_mbid : for mbids
- <entity>_msid : for msids
- <entity>_name : for the text value of one of the above.

Reply:
thanks. I updated this to be releasembid